lib/verdict/experiment.rb

```diff
@@ -35,6 +36,10 @@ def store_unqualified?
     @store_unqualified
   end
 
+  def timestamps_out_of_band?
```

Comment:
Hmm...I'm not a fan of this name. Could we use something like `manual_timestamp?` or `auto_timestamp` (with reversed logic)?